docs/flutterbloccoreconcepts.md

```diff
@@ -245,6 +249,10 @@ RepositoryProvider(
 then from `ChildA` we can retrieve the `Repository` instance with:
 
 ```dart
+// with extensions
+context.repository<RepositoryA>(context);
```

Comment:
I know people originally requested `context.repo<RepositoryA>(context)` so I would love feedback.

Reply:
Abbreviations are usually a sign of bad naming conventions. I'd go with `repository`. There's also intellisense to help the lazy ones typing the whole thing out ✌ The only good positive thing I see for going with `repo` is if someone's using a max line length of 80, it'd help out with not breaking the line.

Reply:
+1 to the full `repository`... you will get autocomplete support from the IDE anyways, so better be verbose in an SDK IMO